llvm/test/CodeGen/RISCV/misched-mem-clustering.mir

```diff
@@ -0,0 +1,64 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py UTC_ARGS: --version 5
+# RUN: llc -mtriple=riscv64 -x mir -mcpu=sifive-p470 -mattr=+use-postra-scheduler -verify-misched -enable-post-misched=true \
+# RUN:     -riscv-misched-load-store-clustering=false -debug-only=machine-scheduler \
+# RUN:     -start-before=machine-scheduler -stop-after=postmisched -o - 2>&1 < %s \
+# RUN:   | FileCheck -check-prefix=NOCLUSTER %s
+# RUN: llc -mtriple=riscv64 -x mir -mcpu=sifive-p470 -mattr=+use-postra-scheduler -verify-misched -enable-post-misched=true \
+# RUN:     -debug-only=machine-scheduler \
+# RUN:     -start-before=machine-scheduler -stop-after=postmisched -o - 2>&1 < %s \
+# RUN:   | FileCheck -check-prefix=MEMCLUSTER %s
```

Comment:
After adding the option above, please add a RUN with postra scheduling enabled but no postra clustering.